### PR TITLE
Add visual-diff setup and instructions

### PR DESCRIPTION
**Pull request by margaree**
Will close https://github.com/BrightspaceUI/lit-element-template/issues/7

I made it non-optional (e.g., not included in the setup prompts).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ To do cross-browser testing using Sauce Labs, the API key needs to be encrypted
 
 To learn more about how to set this up, see the [testing](https://github.com/BrightspaceUI/guide/wiki/Testing) section of The Guide.
 
+### Visual Diff Testing
+
+To do visual diff testing, the `GITHUB_RELEASE_TOKEN`, `VISUAL_DIFF_S3_ID`, and `VISUAL_DIFF_S3_SECRET` need to be encrypted into the .travis.yml file.
+
+To learn more about how to set this up, see the [Running in CI](https://github.com/BrightspaceUI/visual-diff#running-in-ci) section of the visual-diff readme.
+
+To use the Visual Difference Bot to run visual diff testing, see [here](https://github.com/BrightspaceUI/visual-difference-bot/blob/master/README.md/#utilizing-the-deployed-bot-for-your-existing-brightspace-repo).
```
Comment (margaree) [inline README.md]: FYI @DiljotSG

Comment (DiljotSG) [reply README.md]: It looks good to me!

File: package.json [hunk 2]
```diff
@@ -8,20 +8,24 @@
...
+    "test:diff": "mocha ./**/*.visual-diff.js -t 40000",
+    "test:diff:golden": "mocha ./**/*.visual-diff.js -t 40000 --golden",
+    "test:diff:golden:commit": "commit-goldens",
     "test:headless": "karma start",
     "test:sauce": "karma start karma.sauce.conf.js"
   },
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@brightspace-ui/visual-diff": "^1.0.1",
```
Comment (dlockhart) [inline package.json]: Can this just be `^1`?

File: test/_element.visual-diff.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+			import '../<%= shortName %>.js';
+		</script>
+		<title><%= name %> visual-diff tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+	</head>
+	<body class="d2l-typography">
+		<div class="visual-diff">
+			<<%= name %> id="<%= shortName %>"></<%= name %>>
```
Comment (dlockhart) [inline test/_element.visual-diff.html]: The `id` typically maps to a state to test. So here it could be "default" or something?

**Pull request merged**